### PR TITLE
Feature: RefinementAutosuggest for perf

### PR DESCRIPTION
**Pull request by ssetem**
Thanks for using `Searchkit`
I notice some performance improvements that could be made to buildhub

- Updated to latest searchkit 2.3.0-9
- Use `RefinementAutosuggest` components instead of `RefinementListFilter`
- Improves performance drastically, as not needing to load few thousand facets

ezgif-5-2ebb060900

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -225,47 +229,42 @@ class App extends Component {
 
             <LayoutBody>
               <SideBar>
-                <RefinementListFilter
+                <RefinementAutosuggest
                   field="target.version"
-                  title="Version"
+                  title="Version"
```
Comment (magopian) [inline ui/src/App.js]: It seems there's trailing blank spaces

File: ui/src/index.css [hunk 2]
```diff
@@ -14,11 +14,11 @@ body {
   padding: 20px 0 20px 15px;
 }
 
-.sk-panel__content {
+/* .sk-panel__content {
   max-height: 300px;
   overflow-y: auto;
   padding-right: 10px;
-}
+} */
```
Comment (magopian) [inline ui/src/index.css]: If it's not used anymore, I'd rather have it removed entirely ;)

**Comment by magopian** [comment]
What do you think @Natim ?

**Comment by ssetem** [comment]
Could still be explorable when focusing on empty box, it shows 20 or so facets, ordered by count perhaps?

**Comment by Natim** [comment]
@magopian When entering the select box it seems that it shows some possibilities even before taping, doesn't it?

File: ui/src/App.js [hunk 3]
```diff
@@ -225,47 +229,42 @@ class App extends Component {
...
+                  orderKey="_term"     
+                  multi={true}                               
                 />
-                <RefinementListFilter
+                <RefinementAutosuggest
                   field="target.locale"
                   title="Locale"
                   id="locale"
-                  size={1000}
+                  size={10}
```
Comment (Natim) [inline ui/src/App.js]: Can we make it 20 or 50? ordered by number of entries is a great idea I guess.

Comment (ssetem) [reply ui/src/App.js]: yeah sure, I'l make the fixes and post another interaction gif

**Comment by ssetem** [comment]
Suggest results are now ordered by count and set to 20

ezgif-5-4e1b51de15

**Comment by ssetem** [comment]
@magopian @Natim 
I just deployed to free heroku node if you need to preview
https://buildhub-suggest.herokuapp.com/

**Comment by Natim** [comment]
Now that I see it I think that for versions it makes more sense to keep the alphabetical order.
For the rest I think it is good now 👍

**Comment by magopian** [comment]
@Natim who could we ask to test the usability? We're not the end users, so I'd rather not take a decision without at least letting them know ;)

**Comment by ssetem** [comment]
yeah, no rush to merge this, it can even live in a WIP branch. this was more to incubate a potential improvement.

**Comment by peterbe** [comment]
I have tested this locally after rebasing it. It worked. It's definitely snappier and feels better. 

Also, I didn't want this to rot. Especially since I intend to [swap out `npm` for `yarn`](https://github.com/mozilla-services/buildhub/issues/365).

**Pull request merged**